spec/features/roles_spec.rb

```diff
@@ -28,8 +28,10 @@
     end
 
     scenario 'allows resource managers to manage resources' do
-      visit '/'
-      expect(page).to have_content 'Welcome'
+      visit users_path
```

Comment:
We're going to see the benefits of page objects now these are getting more meaningful. Perhaps we should introduce SitePrism?